internal/handler/v2/delegations.go

```diff
@@ -0,0 +1,46 @@
+package v2
+
+import (
+	"github.com/valyala/fasthttp"
+)
+
+// DelegationsByAddress handles GET "/v2/delegations/{address}".
+// Queries Numia and returns all delegations for a given address.
+// Accepts both bech32 and hex addresses.
+// Returns
+// [
+//
+//	{
+//	  "validatorAddress": "evmosvaloper1....",
+//	  "delegated": {
+//	    "denom": "aevmos",
+//	    "amount": "5000....."
+//	  },
+//	  "unclaimed": {
+//	    "denom": "aevmos",
+//	    "amount": "5000....."
+//	  }
+//	}
+//
+// ]
+func (h *Handler) DelegationsByAddress(ctx *fasthttp.RequestCtx) {
+	address := ctx.UserValue("address").(string)
+	// TODO - validate address before querying numia
+	if address == "" {
+		ctx.SetStatusCode(fasthttp.StatusBadRequest)
+		errorResponse := &ErrorResponse{
+			Error: "Missing address",
+		}
+		sendJSONResponse(ctx, errorResponse)
+		return
+	}
+
+	delegations, err := h.numiaRPCClient.QueryDelegations(address)
```

Comment:
Q: should we retry the query if fails? maybe we can use a retryable HTTP client, e.g. [this lib](https://github.com/hashicorp/go-retryablehttp), or add retry on our existing code

Reply:
I think thats a great idea! Can you create a ticket and assign to it to me? I can tackle that on a separate PR once back

Reply:
created a ticket for this